validateIntrospectorScript.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+#
+# Copyright (c) 2022, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+found_error=0
+
+INTROSPECT_DOMAIN_SCRIPT_FILE=$(find "$(pwd -P)" -type f -name "introspectDomain.sh" | tail -n 1)
+DOMAIN_INTROSPECTION_COMPLETE_STRING=$(grep -Er "DOMAIN_INTROSPECTION_COMPLETE" * | grep ProcessingConstants.java | grep -v html | sed 's/.*= //g' | sed 's/;$//g')
```

Comment:
(1) Can this also use a hard coded exact location?

(2) Also, it should have a sanity check that the returned value isn't empty. (Or perhaps better - simply check taht the returned value matches the expected "Domain introspection complete." value.)

Reply:
I added a sanity check to make sure that the returned value isn't empty. I was thinking that someone can change the "Domain introspection complete."  string in both ProcessingConstants.java and introspectDomain.java  at the same time and the check shouldn't fail in this case.

Reply:
It's simpler and a little more robust IMO to simply check each of three exact locations (no "find") for the exact expected string. If someone needs to change the string, they can change this test file in addition to the three locations...

Reply:
Changed in [4fd5c6f](https://github.com/oracle/weblogic-kubernetes-operator/pull/3408/commits/4fd5c6f8faa8ddd5711b43372a6c1abdb8a6222f).

Reply:
LGTM